MinecraftClient not found in 1.20.1

When attempting to load `ClientScheduler` 
`Caused by: java.lang.ClassNotFoundException: net.minecraft.class_310
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)`

--- a/gradle.properties
+++ b/gradle.properties
@@ -3,15 +3,18 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 	# check these on https://modmuss50.me/fabric.html
-	minecraft_version=1.18
-	yarn_mappings=1.18+build.1
-	loader_version=0.16.0
+
+minecraft_version=1.18.2
+yarn_mappings=1.18.2+build.4
+loader_version=0.16.10
+
+# Fabric API
 
 # Mod Properties
-	mod_version = 1.0.0
+	mod_version = 1.0.2
 	maven_group = dev.drtheo
 	archives_base_name = scheduler
 
 # Dependencies
 	# check this on https://modmuss50.me/fabric.html
-	fabric_version=0.44.0+1.18
+fabric_version=0.77.0+1.18.2

